device/isis/tlvreplay.yaml

```diff
@@ -0,0 +1,17 @@
+components:
+  schemas:
+    Isis.TlvReplay:
+      description: >-
+        Specification ISIS Tlv Replay Bytes.
+      type: object
```

Comment:
Specification ISIS Tlv (in Bytes) that will be replayed by ISIS router in XXX message/pdu.